soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -515,6 +517,7 @@ void SetAreas(){
     for (auto& loc : region.locations){
```

Comment:
` std::vector<RandomizerRegion> alreadyChecked = {(RandomizerRegion)c};`
It's better to use static_cast for such conversions.
Also better name for 'c' is 'regionType'.